tasks/st0305_support_script_and_upload.rb

```diff
@@ -21,7 +24,11 @@ def puppet_enterprise_support(logage, scope, ticket)
   }
 end
 
-results = {}
+deprecation_msg = "This task is deprecated with support script uploading to be handled manually, and will be removed in a future release.
```

Comment:
Its not manually its just done on the command line rather than the UI, so "according to documentation" is a better choice of phrase